docs/development.md

```diff
@@ -172,4 +172,4 @@ helm get manifest akri | less
 ```
 
 ### Helm Upgrade
-To modify a Akri installation to reflect a new state, you can use [`helm upgrade`](https://helm.sh/docs/helm/helm_upgrade/). See the [modifying a Akri installation document](./modifying-akri-installation.md) for further explanation. 
\ No newline at end of file
+To modify a Akri installation to reflect a new state, you can use [`helm upgrade`](https://helm.sh/docs/helm/helm_upgrade/). See the [modifying a Akri installation document](./modifying-akri-installation.md) for further explanation. 

```

Comment:
nit: should this be an Akri installation?

Reply:
agree ... will fix